src/backend/singular_value_decomposition.py

```diff
@@ -1,8 +1,8 @@
-from backend.database_connection import DatabaseConnection
-from backend.histogram_of_gradients import HistogramOfGradients
+from database_connection import DatabaseConnection
+from histogram_of_gradients import HistogramOfGradients
```

Comment:
Can you try writing

from . import database_connection, histogram_of_gradients

instead of

from database_connection import DatabaseConnection
from histogram_of_gradients import HistogramOfGradients
?
If it works let's use this pattern.

Reply:
This doesn't work... need to create a package for that.... good to keep it this way